symengine/polys.cpp

```diff
@@ -245,8 +246,54 @@ RCP<const Polynomial> neg_poly(const Polynomial &a) {
 }
 
 #ifdef HAVE_SYMENGINE_PIRANHA
-RCP<const Polynomial> mul_poly(RCP <const Polynomial> a, RCP <const Polynomial> b) {
-    hash_set C, A = a->polys_set_, B = b->polys_set_;
+void _normalise_polymul(RCP <const Polynomial> &p, RCP <const Polynomial> &q, vec_symbol vars_,
```

Comment:
Is this the right way of passing polynomials to this function?
@shivamvats @certik